### PR TITLE
update mysql connection string to take env vars

### PR DESCRIPTION
**Pull request by dcordz**

### CODE REVIEW

File: db/mysql.js [hunk 1]
```diff
@@ -13,14 +13,20 @@ hooker.hook(require('mysql/lib/Connection').prototype, 'query', {
...
+		host: process.env.MYSQL_HOST || 'localhost',
+		user: process.env.MYSQL_USER || 'root'
+	});
+}
 
 // pretend we already connected
 connection._connectCalled = true;
 
+// TODO: rename this function?
 exports.register = function mongo(server, options) {
```
Comment (dcordz) [inline db/mysql.js]: Should this be named `mongo`?

Comment (dcordz) [inline db/mysql.js]: Should this function be named `mongo`?

Comment (bizob2828) [reply db/mysql.js]: yep, copypasta fail.

File: db/mysql.js [hunk 2]
```diff
@@ -13,14 +13,20 @@ hooker.hook(require('mysql/lib/Connection').prototype, 'query', {
 	}
 });
 
-const connection = mysql.createConnection({
-	host: 'localhost',
-	user: 'root'
-});
+let connection;
+if (process.env.MYSQL_URI) {
```
Comment (bizob2828) [inline db/mysql.js]: this app doesn't actually connect to mysql. if you look above we're instrumenting query to return what you pass. s you don't really need this.  were you trying to add mysql to the build or something?

File: db/mysql.js (hunk 1 above)
Comment (bizob2828) [reply db/mysql.js]: yea you can rename

Comment (bizob2828) [reply db/mysql.js]: also this repo is wonky. when you merge this make sure you merge to `master`, `hapi18` and `hapi17` they are 3 diff targets. master is latest hapi, hapi18 is obv 18 and so on

**Comment by bizob2828** [comment]
This is no longer relevant

**Pull request closed without merging**